src/index.html

```diff
@@ -12,6 +12,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link rel="stylesheet" href="assets/fontawesome6/css/all.min.css">
+  <!-- <link rel="stylesheet" href="assets/bootstrap.min.css"> -->
```

Comment:
what does this actually do/why? Doesn't appear to be in master/index.html? Appears to be a redundant line

Reply:
Good catch. Old leftover